### PR TITLE
[Feature] Packaging

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

This is a feature branch containing the packaging feature: 

- enable/disable conference calling
- inform users when conference calling feature is enabled

## Notes

https://wearezeta.atlassian.net/browse/SQSERVICES-561

### CODE REVIEW

File: Source/Model/FeatureConfig/Feature.swift [hunk 1]
```diff
@@ -127,8 +148,8 @@ public class Feature: ZMManagedObject {
     ///     - changes: A closure to mutate the fetched instance.
 
     public static func updateOrCreate(havingName name: Name,
-                                      in context: NSManagedObjectContext,
-                                      changes: @escaping (Feature) -> Void) {
+                               in context: NSManagedObjectContext,
+                               changes: @escaping (Feature) -> Void) {
```
Comment (johnxnguyen) [inline Source/Model/FeatureConfig/Feature.swift:152]: Indentation needs fixing.

**Review by typfel — APPROVED** [review]

File: Source/Model/FeatureConfig/Feature.swift (hunk 1 above)
Comment (KaterinaWire) [reply Source/Model/FeatureConfig/Feature.swift]: Done

**Review by johnxnguyen — APPROVED** [review]

**Comment by KaterinaWire** [comment]
@zenkins release major

**Comment by typfel** [comment]
@zenkins release major

**Pull request merged**